go.mod

```diff
@@ -2,4 +2,16 @@ module github.com/caddyserver/nginx-adapter
 
 go 1.13
 
-require github.com/caddyserver/caddy/v2 v2.0.0-beta7
+require (
+	github.com/caddyserver/caddy/v2 v2.0.0-beta12
+	github.com/cenkalti/backoff/v3 v3.2.1 // indirect
+	github.com/miekg/dns v1.1.27 // indirect
+	go.uber.org/atomic v1.5.1 // indirect
+	go.uber.org/multierr v1.4.0 // indirect
+	go.uber.org/zap v1.13.0 // indirect
+	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17 // indirect
+	golang.org/x/sys v0.0.0-20200107162124-548cf772de50 // indirect
+	golang.org/x/tools v0.0.0-20200110042803-e2f26524b78c // indirect
+)
+
+replace github.com/caddyserver/caddy/v2 => ../caddy
```

Comment:
remove this when caddyserver/caddy#2997 is merged (if it's decided to be merged) and update Caddy version in this module

Reply:
Feel free to pin to the commit SHA for now, we can update it later.

Reply:
I did:

`go get github.com/caddyserver/caddy/v2@98bbc54fdcd05a2f53c10e81be7cf94d1dc62522` 

and go.mod now says:

`github.com/caddyserver/caddy/v2 v2.0.0-beta9.0.20200208193637-98bbc54fdcd0`

I'm confused 🤔

Reply:
Ah... that's annoying. I'd claim it's a bug in the Go tool but they would disagree since it's doing lexicographical sorting... sigh.

Anyway, Just replace `beta9` with `beta.13` and it should be fine

Reply:
Oh well, the Go compiler is happy and works fine.

Reply:
Done. Changed it manually as you suggested and nothing broke 😄